index.html

```diff
@@ -886,7 +901,136 @@ <h4 id="event-handlers"><span class="secno">6.3.4 </span>
         </section>
       </section>
       <section>
-        <h3 id="interface-navigatorpresentation"><span class="secno">6.4 </span>
+        <h3 id="interface-availability"><span class="secno">6.4 </span>
+          Interface <a href="#availability"><code>Availability</code></a>
+        </h3>
+        <pre class="idl">interface <dfn id="availability">Availability</dfn> : EventTarget {
+  readonly attribute boolean current;
```

Comment:
I wanted to avoid calling it ```available``` since it lives in the ```Availability``` interface :) however it make more sense if you consider using it like 

```javascript
availability.onchange = function() {
  handleAvailabilityChange(this.available); // vs. this.current or this.value
};
```

Otherwise I agree that ```value``` is a common name used along with ```onchange```.